util/validators.ts

```diff
@@ -32,6 +32,22 @@ export const validateApiList = (val: string): string => {
   return val;
 };
 
+export const validateProductionApiList = (val: string): string => {
+  let result: boolean;
+  try {
+    const apis = val.split(',');
+    result = apis.every(api => PRODUCTION_REQUEST_API_LIST.includes(api));
+  } catch {
+    throw new Error('it was unable to process the provided data');
+  }
+
+  if (!result) {
+    throw new Error('invalid apis in list');
+  }
```

Comment:
i'd like to tweak this code (in the current/shared function) for slightly better feedback, with the invalid values.

```suggestion
  let invalidAPIs: string[];
  try {
    const apis = val.split(',');
    invalidAPIs = apis.filter(api => !API_LIST.includes(api));
  } catch {
    throw new Error('it was unable to process the provided data');
  }

  if (invalidAPIs.length > 0) {
    throw new Error(`invalid apis in list: ${invalidAPIs.join(', ')}`);
  }
```

it'd be helpful in quickly checking issues like this in development or for debugging in production should something like this slip out.